core/package/index.js

```diff
@@ -30,6 +33,12 @@ function shallowCopy(json) {
   }, {});
 }
 
+// determine if a package-lock.json file exists
```

Comment:
This isn't the appropriate place for lockfile maintenance. It's a function of the version command, not integral to the nature of a Package.